app.js

```diff
@@ -159,6 +159,7 @@ var logger = new Logger({
 	echo: process.env.LOG_LEVEL || appConfig.consoleLogLevel,
 	errorLevel: process.env.FILE_LOG_LEVEL || appConfig.fileLogLevel,
 	filename: appConfig.logFileName,
+	network: appConfig.network,
```

Comment:
Suggest not to add network specific dependency to logger. It's against of the separation of concern rule. Logger just need to log, for it it needs a file path, log level and any additional config attribute for logging. 

The logic to choose the log file path based on network is responsibility of one level higher.